presto-main/src/main/java/com/facebook/presto/testing/LocalQueryRunner.java

```diff
@@ -566,6 +568,7 @@ private LocalQueryRunner(Session defaultSession, FeaturesConfig featuresConfig,
                 .put(DropMaterializedView.class, new DropMaterializedViewTask())
                 .put(RenameColumn.class, new RenameColumnTask())
                 .put(RenameTable.class, new RenameTableTask())
+                .put(RenameView.class, new RenameViewTask(metadata, accessControl))
```

Comment:
The same as mentioned above, is it necessary to pass `metadata` and `accessControl` on construction of `RenameViewTask`?